docs/layouts/shortcodes/generated/sql_client_configuration.html

```diff
@@ -8,12 +8,6 @@
         </tr>
     </thead>
     <tbody>
-        <tr>
```

Comment:
Can we remove the documentation when the configuration is totally dropped? I think it is helpful to highlight this configuration is deprecated and please use "table.display.max-column-width" instead. This can make the migration smoother.

Reply:
I have the same concern as I first saw this change. It is a generated content based on the current generation rule. A modification requires a separate PR to upgrade the generation rule and logic.

Reply:
As example of the current generation rule: ExecutionConfigOptions#TABLE_EXEC_LEGACY_TRANSFORMATION_UIDS is deprecated and not dropped. Its content has been removed since 1.15: https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/config/#table-exec-uid-generation

Reply:
If the generation rule is based on the `@Deprecated` annotation, we can add the deprecated message in this `withDescription` field and remove the `@Deprecated` annotation (add in the future once the rule is adapted).

Reply:
fixed, new ticket has been created: FLINK-30862.